lib/resty/chash.lua

```diff
@@ -38,12 +38,32 @@ void chash_point_delete(chash_point_t *old_points, uint32_t old_length,
     uint32_t id);
 ]]
 
+local new_tab
+do
+    local ok
+    ok, new_tab = pcall(require, "table.new")
+    if not ok or type(new_tab) ~= "function" then
+        new_tab = function (narr, nrec) return {} end
+    end
+end
+
+local function _copy(nodes)
```

Comment:
The `_copy` should go inside the `local copy` block.

Reply:
already move this code to utils module